docs/upstream.md

```diff
@@ -64,12 +69,12 @@ $ git reset --hard upstream/master
 Just to make sure that all things are synced correctly, run the following:
 
 ```bash
-$ git checkout main
+$ git checkout master
 
 $ git fetch upstream
 $ git merge upstream/master
 ```
 
 ## Maintain upstream
 
-Once you have connected your local forked clone to the upstream repo, maintenance is the key. It is important to ensure that your `master` branch is always updated BEFORE you create a new branch intended to be used with a new pull request. This will help to eliminate any potential issues with merging when the `master` branches are out of sync.
+Once you have connected your local forked clone to the upstream repo, mastertenance is the key. It is important to ensure that your `master` branch is always updated BEFORE you create a new branch intended to be used with a new pull request. This will help to eliminate any potential issues with merging when the `master` branches are out of sync.
```

Comment:
Please be careful with search and replace. You replaced `maintenance` with `mastertenance`.